hub/features/image.py

```diff
@@ -1,17 +1,30 @@
 from typing import Tuple
 
-import hub.features.tensor as tensor
+from hub.features.features import Tensor
 
 
-class Image(tensor.Tensor):
-    def __init__(
-        shape: Tuple[int, ...] = None,
-        dtype=None,
-        encoding_format: str = None,
-        channels=None,
-    ):
-        super().__init__(shape, dtype)
+class Image(Tensor):
+    def __init__(self,
+                 shape: Tuple[int, ...] = None,
+                 dtype = None,
+                 encoding_format: str = None,
+                 channels = None,
+    ):        
+        self._set_dtype(dtype)
+        super(Image, self).__init__(shape, dtype)
+        self._set_channels(channels)
```

Comment:
I think shape should contain channels actually, because we inherit from tensor.
@davidbuniat you opinion would be helpful here too.

Reply:
I think so as well, channels should be included in shape, this could be optional for setting names of channels ("nir", "red", "green",...)